### DISCUSSION TITLE
Not getting all matches in a repo

### DISCUSSION THREAD

**Discussion by akihironitta** [community/community#10035] [Other Feature Feedback, Questions, & Ideas] [answered]
I am trying to get all occurrences of `restore_loops` in a repo I'm working on, but search results don't seem to include all of them as I grepped the codebase locally.

### 2 results in Code Search

<img width="752" alt="image" src="https://user-images.githubusercontent.com/20610905/149146046-0e9d696d-ecd3-4044-8f5f-c07cd717ea25.png">

search results: https://cs.github.com/PyTorchLightning/pytorch-lightning?q=%22restore_loops%22
the top one in the results: https://cs.github.com/PyTorchLightning/pytorch-lightning/blob/6107ce8e0d2feaed0263c0a60fc6c031603fd9ea/pytorch_lightning/trainer/connectors/checkpoint_connector.py?q=%22restore_loops%22#L212

### 3 results from grep
```console
$ grep -rnI restore_loops
pytorch_lightning/trainer/connectors/checkpoint_connector.py:198:        self.restore_loops()
pytorch_lightning/trainer/connectors/checkpoint_connector.py:212:    def restore_loops(self) -> None:
tests/trainer/connectors/test_checkpoint_connector.py:185:            trainer.checkpoint_connector.restore_loops()
```

---

I'm sorry if there's any ongoing issue already, and thank you so much for this amazing feature!!! :]

**Comment by akihironitta** [comment, marked as answer]
Just realised that multiple matches in a single file are grouped into one result :sweat_smile: :

<img width="139" alt="image" src="https://user-images.githubusercontent.com/20610905/149147309-b5d0256c-5e7a-4472-b5bd-bcaeb18baf47.png">

**Reply by look** [reply]
You're right, there is a subtle indication that there are more matches in that file. We try to show you the best match in a file, but I'll mark this down as feedback requesting a way to see more of (or all of) the matches.

**Reply by akihironitta** [reply]
Thanks for the follow-up. I definitely prefer seeing all of them to seeing only one of the results per file.
